### PR TITLE
Run update_deps on the release-1.13 branch

### PR DESCRIPTION
**Pull request by lei-tang**
**Please provide a description of this PR:**
For Istio 1.13.6 release, run update_deps on the release-1.13 branch.

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [X ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by stevenctl** [comment]
Do we need to update deps/was this done in other releases yet? Looks like there are some api changes that need to be fixed

**Comment by stevenctl** [comment]
```
operator/pkg/apis/istio/v1alpha1/common.go:57:14: invalid operation: iops.Values[globalKey] (type *structpb.Struct does not support indexing)
```

**Comment by stevenctl** [comment]
I think #37867 may have been the PR that resolved this. That was on master before 1.14 came. We may want to selectively not update a few deps

**Comment by stevenctl** [comment]
or cherry-pick a bunch of large changes... i'm more comfortable with the former.

File: go.mod [hunk 1]
```diff
@@ -62,41 +62,41 @@ require (
...
 	gopkg.in/square/go-jose.v2 v2.6.0
 	gopkg.in/yaml.v2 v2.4.0
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.8.0
-	istio.io/api v0.0.0-20220718214027-666b55ffb5a8
-	istio.io/client-go v1.13.5-0.20220718214527-fcd2b1530308
-	istio.io/pkg v0.0.0-20220413180906-765b7512325f
-	k8s.io/api v0.23.3
+	istio.io/api v0.0.0-20220718195626-87dfc8c8c07a
```
Comment (ericvn) [inline go.mod:88]: Something is wrong with the update_deps as it's pulling the incorrect version of the other repos. This commit is from the master branch of istio/api for example. It should be the latest from the 1.13 branch. I think there is a env var that the script will use to get from the appropriate branch. I suspect it's default is master.

**Comment by ericvn** [comment]
https://github.com/istio/istio/blob/release-1.13/bin/update_deps.sh#L19 is the issue. Either set UPDATE_BRANCH before calling the script or update the script in this branch to call out the release-1.13.

**Comment by ericvn** [comment]
See https://github.com/istio/istio/blob/release-1.13/bin/update_deps.sh#L19. It defaults to master. Either change the default or set UPDATE_BRANCH before calling the script.

**Pull request merged**